tests/vcr_python_wrapper.py

```diff
@@ -10,17 +10,17 @@
 # We need our own json level2 matcher, because, python2 and python3 do not save
 # dictionaries in the same order
 def body_json_l2_matcher(r1, r2):
-    if r1.headers.get('content-type') == 'application/json' and r2.headers.get('content-type') == 'application/json':
+    if r1.headers.get('content-type') == r2.headers.get('content-type') == 'application/json':
         if r1.body is None or r2.body is None:
             return r1.body == r2.body
         body1 = json.loads(r1.body.decode('utf8'))
         body2 = json.loads(r2.body.decode('utf8'))
-        if 'search' in body1:
-            body1['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body1['search'])))
-        if 'search' in body2:
-            body2['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body2['search'])))
+        if 'common_parameter' in body1 and 'common_parameter' in body2:
+            if body1['common_parameter'].get('parameter_type') == body2['common_parameter'].get('parameter_type') in ['hash', 'json', 'yaml']:
+                body1['common_parameter']['value'] = json.loads(body1['common_parameter'].get('value'))
+                body2['common_parameter']['value'] = json.loads(body2['common_parameter'].get('value'))
         return body1 == body2
-    elif r1.headers.get('content-type') == 'multipart/form-data' and r2.headers.get('content-type') == 'multipart/form-data':
+    elif r1.headers.get('content-type') == r2.headers.get('content-type') == 'multipart/form-data':
```

Comment:
I find it interesting that this works here, as in #382 I had to use `startswith` as the header was something like `multipart/form-data; boundary=44d0d62903f617236943e867fe494c96`

Reply:
Right, but i did not really change anything here.
Maybe we should not question vcr...

Reply:
Yeah, not saying you did, just an observation :)